### ISSUE TITLE
instantiating a filepond instance with allow-remove=false sets allow-remove=false for all future instances

### ISSUE THREAD

**Issue by shasderias** [pqina/vue-filepond#199]
**Describe the bug**
When instantiating multiple filepond instances, once a filepond instance is instantiated with allow-remove=false, future filepond instances will also be instantiated with allow-remove=false.

**Logs**
N/A

**To Reproduce/Expected behavior**
https://codesandbox.io/s/happy-resonance-7cr9v

1. Open codesandbox link
2. State 0 FilePond works as expected, remove button is shown, initial files can be removed
3. Click toggle, State 1 FilePond is instantiated (with allow-remove=false)
4. State 1 FilePond also works as expected, remove button is not shown, initial files cannot be removed
5. Click toggle, State 0 FilePond is instantiated

Expected: remove button to be shown for State 0 FilePond
Got: remove button not shown

**Stacktraces**
N/A

**Information about your project:**
- Your browser and the version: (e.x. Chrome 52.1, Firefox 48.0, IE 11)
Version 89.0.4389.114 (Official Build) (64-bit)

- Your operating system: (e.x. MacOS 10, Ubuntu Linux 19.10, Windows 10, etc)
Windows 10

- FilePond version (Please check you can reproduce the issue with the latest release!)
4.27.0

**Additional context**
Possibly related to pqina/filepond#613?

**Comment by rikschennink** [comment]
Thanks for reporting, weird, I suspect it has to do with this line, but not sure, PR would be appreciated.
https://github.com/pqina/vue-filepond/blob/master/lib/index.js#L137

**Comment by vesper8** [comment]
I notice the same is true for other settings. For example I set the imagePreviewHeight = 128 on one instance where I want a 2-column grid displayed

And then on another instance where I want a 3-column grid I set it as imagePreviewHeight = 100

But it still uses the 128, it only respects the 100 if I manually reload the page

Is there any workaround for this? Is there a way to force a destroy/re-render so this doesn't happen?

@rikschennink @shasderias ?
